packages/salesforcedx-utils-vscode/src/workspaces/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
With webpack we make the assumption that a module's entry point will be its index.ts file.